### ISSUE TITLE
[BUG_SP] Kursgenerator hat Probleme mit Inselumfahrung

### ISSUE THREAD

**Issue by 0scarZulu** [Courseplay/Courseplay_FS22#812]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
Kurs generieren auf einem Feld wo Feldinseln sind

**Actual behavior**
Nach einer Kursgenerierung geht das Vorgewende durch die Feldinsel durch obwohl sie zuvor sauber umfahren wurde auf den Bahnen.

**Expected behavior**
Auch im Vorgewende wird die Feldinsel sauber umfahren 

**Screenshots**
![Screenshot_1](https://user-images.githubusercontent.com/78980660/151933601-7ea49848-de06-4515-ade2-b03598b5717e.png)

**Courseplay version**
Version used: 7.0.0.25

**Log**
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/7976829/log.txt)

**Map**
FS22 Goliszew

**Vehicles/ implements involved**
Traktor und Zetter
Mod: Ja

**Relevant script mods**
nicht für diese Sache relevant

**Comment by pvaiko** [comment]
Bitte Fields exportieren und hier einfügen, siehe (https://github.com/Courseplay/Courseplay_FS22/wiki#how-to-post-your-field-definitions)

**Comment by 0scarZulu** [comment]
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/7996844/log.txt)
Will er nicht. Bringt nen LUA Fehler dann.

**Comment by pvaiko** [comment]
Ja, Feld 38 hat ein Problem, wahrscheinlich zu nah an anderen Felder dran.

**Comment by Tensuko** [comment]
Should work now in 7.0.0.28

**Comment by 0scarZulu** [comment]
@Tensuko äh nope :) 
![image](https://user-images.githubusercontent.com/78980660/153158884-80d8f16e-58c0-419f-817e-24da8dc9aea8.png)

Das Feld hat er vor der 7.0.0.28 zB korrekt berechnet. Und jetzt macht er da ein Quatsch :D

**Comment by pvaiko** [comment]
Dafür muss jetzt cpSaveAllFields funktionieren. Und ich möchte wirklich gerne den Feldrand sehen wenn du in AI Screen das Feld auswählst.

**Comment by pvaiko** [comment]
(Ausserdem den Startpunkt gerade in diese Ecke zu setzen ist schlicht unfair...)

**Comment by 0scarZulu** [comment]
Gut dann hier mal mit verändertem Startpunkt und der cpSaveAllfields Datei
[cpFields.zip](https://github.com/Courseplay/Courseplay_FS22/files/8032798/cpFields.zip)
![Screenshot_1](https://user-images.githubusercontent.com/78980660/153204172-0641f5d5-e8c6-41cf-a2e3-cabb7b7fd8d6.png)
![Screenshot_2](https://user-images.githubusercontent.com/78980660/153204179-2e126c0f-bd57-4c52-8f84-14a8e77fd80a.png)

**Comment by pvaiko** [comment]
Ja, das ist eine bekannte Enschränkung, wie es auch im Log steht:
`Island 2: Course has turn start or end waypoints between 530-538, no bypass`

Vielleicht wenn wir irdendwann nicht mehr wichtigeres zu tun haben ...
